client/src/components/settings/appearance-management/element-preview.js

```diff
@@ -16,13 +16,24 @@
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import {Button, Alert, Input} from 'antd';
+import {Button, Alert, Input, Table, Select, Icon} from 'antd';
 import Menu, {MenuItem} from 'rc-menu';
 import classNames from 'classnames';
 import {sectionNames} from './utilities/variable-sections';
 import styles from './element-preview.css';
 
 class ElementPreview extends React.Component {
+  constructor (props) {
+    super(props);
+    this.state = {
+      navigationSelected: {
+        home: false,
+        run: false,
+        search: false
+      }
+    };
+  }
+
```

Comment:
Though this is OK in terms of [`constructor` usage in React component](https://reactjs.org/docs/react-component.html#constructor), we do not have any specific initialization logic here (like binding methods). Let's change this to general state initialization:
```javascript
class ElementPreview extends React.Component {
  state = {
    navigationSelected: {
      home: false,
      run: false,
      search: false
    }
  };
}
```
as we do so in other components of the application.